core/include/gnuradio-4.0/Block.hpp

```diff
@@ -1483,15 +1482,15 @@ class Block : public lifecycle::StateMachine<Derived> {
             // See https://github.com/fair-acc/gnuradio4/issues/444
```

Comment:
I think the publishing/forwarding of samples/tags should be not related to how many samples we consumed at the inputs.